src/pcapng/blocks/interface_statistics.rs

```diff
@@ -125,11 +128,21 @@ impl<'a> PcapNgOption<'a> for InterfaceStatisticsOption<'a> {
         Ok(opt)
     }
 
-    fn write_to<B: ByteOrder, W: Write>(&self, writer: &mut W) -> IoResult<usize> {
-        match self {
+    fn write_to<B: ByteOrder, W: Write>(&self, state: &PcapNgState, interface_id: Option<u32>, writer: &mut W) -> Result<usize, PcapError> {
+        Ok(match self {
             InterfaceStatisticsOption::Comment(a) => a.write_opt_to::<B, W>(1, writer),
-            InterfaceStatisticsOption::IsbStartTime(a) => a.write_opt_to::<B, W>(2, writer),
-            InterfaceStatisticsOption::IsbEndTime(a) => a.write_opt_to::<B, W>(3, writer),
+            InterfaceStatisticsOption::IsbStartTime(timestamp) => {
+                writer.write_u16::<B>(2)?;
+                writer.write_u16::<B>(8)?;
+                state.encode_timestamp::<B, W>(interface_id.unwrap(), *timestamp, writer)?;
```

Comment:
I think the constants 2 and 8 should be explained. I think it would be enough to do this:

```suggestion
                let code = 2;
                writer.write_u16::<B>(code)?;
                let length = 8;
                writer.write_u16::<B>(length)?;
                state.encode_timestamp::<B, W>(interface_id.unwrap(), *timestamp, writer)?;
```

Reply:
Thanks